Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetUITest.swift

```diff
@@ -628,7 +628,7 @@ extension PaymentSheetUITest {
         app.buttons["Pay $50.99"].tap()
 
         let successText = app.staticTexts["Success!"]
-        XCTAssertTrue(successText.waitForExistence(timeout: 10.0))
+        XCTAssertTrue(successText.waitForExistence(timeout: 20.0))
```

Comment:
Can we remove the +2 second delay from https://github.com/stripe/stripe-ios/pull/2364/files#diff-2e5b1fdaf1439a692709b9d984e5475e27a0914bb98d482d217772eacb4c8eabR699

Why are these taking > 10 seconds?